pkg/virtctl/vm/remove_volume_test.go

```diff
@@ -127,6 +132,30 @@ var _ = Describe("Remove volume command", func() {
 		}
 	}
 
+	expectVMIEndpointRemoveVolumeErrorFunc := func(dryRun bool, errFunc func() error) {
```

Comment:
Same, can you merge this with the existing func?

Reply:
Done